torch_xla/csrc/ops/ops.h

```diff
@@ -106,7 +106,8 @@ NodePtr AddMatMulOp(const Value& input, const Value& weight, const Value& bias,
 NodePtr Dot(const Value& input, const Value& weight,
             bool use_full_conv_precision);
 
-NodePtr MatMul(const Value& lhs, const Value& rhs);
+NodePtr MatMul(const Value& lhs, const Value& rhs,
+               bool use_full_conv_precision);
```

Comment:
We should not have this flag in the APIs at all.

Reply:
So now we're removing this from everywhere, including nodes? I distinctly remember the discussion when we first added it, that a global flag would be bad. But ok.

Reply:
We just use them when we call XLA.

xla::Dot(x, y, gimme_the_flag());

That is by any means a global thing, are by polluting the APIs we are trying to make it look like a non global, but its value is indeed global.
Better leave the APIs clean. Nobody cares besides tests.
Given that you do the changes, can you use a get/set API for it?

Reply:
Note that I now have to change it *everywhere*, including the `XlaModule`. Do you still think this is a good idea?

Reply:
I love it! :)
Mucho bonus points!

Reply:
Done.